### PR TITLE
CC | Build and use a TDX specific guest image

### PR DESCRIPTION
**Pull request by fidencio**
As we're switching to using EAA KBC with TDX, we should build, distribute, and use an image that supports that.

Fixes: #5563

### CODE REVIEW

**Review by stevenhorsman — APPROVED** [review]
It looks okay to me. Thanks

File: tools/packaging/guest-image/build_image.sh [hunk 1]
```diff
@@ -90,6 +95,7 @@ Options:
  --imagetype=${image_type}
  --prefix=${prefix}
  --destdir=${destdir}
+--image_initrd_sufix=${image_initrd_suffix}
```
[the remark this answers is not in the document]
Comment (fidencio) [reply tools/packaging/guest-image/build_image.sh]: Done, thanks!

**Comment by fidencio** [comment]
Adding the `do-not-merge` label so @arronwy has a chance to review and test it before it gets merged.

**Comment by fidencio** [comment]
Oops, wrong one. :-)

**Pull request closed without merging**